internal/proto/reader.go

```diff
@@ -21,6 +21,8 @@ const (
 
 const Nil = RedisError("redis: nil")
 
+const BusyGroup = RedisError("BUSYGROUP Consumer Group name already exists")
```

Comment:
I don't see this returned from anywhere and TBH I don't think this deserves a separate variable. If you think otherwise please add it in a separate PR.